modules/pytket-honeywell/pytket/extensions/honeywell/backends/honeywell.py

```diff
@@ -61,7 +61,7 @@
 
 HONEYWELL_URL_PREFIX = "https://qapi.honeywell.com/v1/"
 
-HONEYWELL_DEVICE_APIVAL = "HQS-LT-1.0-APIVAL"
+HONEYWELL_DEVICE_APIVAL = "HQS-LT-S1-APIVAL"
```

Comment:
This variable is unused. Is that intended ?

Reply:
no, will remove